libsoftwarecontainer/unit-test/envgatewayparser_unittest.cpp

```diff
@@ -99,7 +99,7 @@ TEST_F(EnvGatewayParserTest, TestValidConfModeSet) {
     ASSERT_EQ(result.second, value);
```

Comment:
Is there any risk that these re-used variables (result and store) are not cleared as they should and can affect other test cases? I understand that it looks a bit cleaner to not create the variables in every test case, but this type of coding style makes me a bit nervous, since the variables are never cleaned/cleared before the test case finishes.